src/lib/pages/frontsite/Home/Sections/SectionInstallation.js

```diff
@@ -0,0 +1,139 @@
+                          here
+                        </a>
+                        .
+                      </List.Item>
+                      <List.Item>
+                        Bootstrap the instance: <br />
+                        <Message>$ ./scripts/bootstrap</Message>
+                      </List.Item>
+                      <List.Item>
+                        Initialize the instance: <br />
+                        <Message>$ ./scripts/setup</Message>
+                      </List.Item>
+                      <List.Item>
+                        Run the server: <br />
+                        <Message>$ ./scripts/server</Message>
+                      </List.Item>
+                    </List>
+                  </Card.Description>
+                </Card.Content>
+              </Card>
+              <Card color="orange" className="install-card">
+                <Card.Content>
+                  <Header size="medium">
+                    <Label circular size="huge" color="orange">
+                      2
+                    </Label>{' '}
+                    Install Frontend
+                  </Header>
+                  <Card.Description>
+                    <List>
+                      <List.Item className="install-item">
+                        Clone the project repository from{' '}
+                        <a href="https://github.com/inveniosoftware/react-invenio-app-ils">
+                          here
+                        </a>
+                        .
+                      </List.Item>
+                      <List.Item>
+                        Install the npm dependencies: <br />
+                        <Message>$ npm install</Message>
+                      </List.Item>
+                      <List.Item>
+                        Run the UI: <br />
+                        <Message>$ npm start</Message>
+                      </List.Item>
+                    </List>
+                  </Card.Description>
+                </Card.Content>
+              </Card>
+              <Card color="orange" className="install-card">
+                <Card.Content>
+                  <Header size="medium">
+                    <Label circular size="huge" color="orange">
+                      3
+                    </Label>{' '}
+                    Visit your browser
+                  </Header>
+                  <Card.Description>
+                    Visit{' '}
+                    <a href="http://localhost:3000/">http://localhost:3000</a>{' '}
+                    in you browser.
+                  </Card.Description>
+                </Card.Content>
+              </Card>
+            </Card.Group>
+            <Grid>
+              <Grid.Column width={16} textAlign="center">
+                <Button
+                  primary
+                  href="https://invenioils.docs.cern.ch/"
+                  className="headline-quick-access"
```

Comment:
`headline-quick-access` does this exist in styles?

Reply:
it's here https://github.com/inveniosoftware/react-invenio-app-ils/blob/170ddbee81145851e0224126e8054db676599a29/src/semantic-ui/site/elements/button.overrides#L38